app/views/skins/_wizard_form.html.erb

```diff
@@ -108,4 +108,28 @@
   <%= hidden_field_tag 'wizard', true %>
 </fieldset>
 
+<fieldset>
```

Comment:
I think the indenting in this new section got a little wonky. For example, it looks like `<legend>` is indented four spaces under `<fieldset>` instead of two, and the closing `</fieldset>` tag doesn't seem to line up with the opening one. 

If you could also update it to use double quotes around strings, that would be fantastic.